how-to/debug-buildfarm-builder.rst

```diff
@@ -0,0 +1,49 @@
+Debug Build Farm
+================
+
+Access a builder in qastaging
+-----------------------------
+
+The builders are ephemeral, meaning they fully reset after a build.
+Therefore, to access the builder, one should either:
+
+* access it during the duration of a build.
+
+* set the builder to ``manual`` after starting a build so that it doesn't
+  automatically trigger the reset when the build finishes.
+
+
+Set a builder to manual
+~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Go to https://qastaging.launchpad.net/builders.
+
+2. Click on the builder you want to set to manual.
+
+3. Click the "Switch to manual-mode" button.
```

Comment:
I cannot see this button, most probably as I do not have admin permissions.

Could you add a note about the required permissions?

And ideally link it to the notes about how to get admin on qastaging @simonepelosi created / wanted to create?

Reply:
That's a great idea! @simonepelosi are you planning on adding those soon? Otherwise, I might just mention that one needs to be part of the `~admins` team, and later when that bit is added, we link it here?

Reply:
Sure, I'm adding those right now. As you suggested in a private conversation I'm refactoring the new-user.rst file adding the new information and re-structuring it :)

Reply:
https://github.com/canonical/launchpad-manual/pull/48

Reply:
Awesome, thanks! I'll update this PR once that one gets merged then